packages/rancher-kiali-server/rancher-kiali-server.patch

```diff
@@ -37,22 +37,24 @@ diff -x '*.tgz' -x '*.lock' -uNr packages/rancher-kiali-server/charts-original/C
 +  catalog.cattle.io/requires-gvr: prometheuses.monitoring.coreos.com/v1
 +  catalog.rancher.io/namespace: cattle-istio-system
 +  catalog.rancher.io/release-name: rancher-kiali-server
++  catalog.cattle.io/hidden: true
 diff -x '*.tgz' -x '*.lock' -uNr packages/rancher-kiali-server/charts-original/templates/_helpers.tpl packages/rancher-kiali-server/charts/templates/_helpers.tpl
 --- packages/rancher-kiali-server/charts-original/templates/_helpers.tpl
 +++ packages/rancher-kiali-server/charts/templates/_helpers.tpl
-@@ -17,10 +17,7 @@
+@@ -17,11 +17,7 @@
    {{- .Values.fullnameOverride | trunc 63 | trimSuffix "-" }}
  {{- else }}
    {{- $name := default .Chart.Name .Values.nameOverride }}
 -  {{- if contains $name .Release.Name }}
 -    {{- .Release.Name | trunc 63 | trimSuffix "-" }}
 -  {{- else }}
 -    {{- printf "%s-%s" .Release.Name $name | trunc 63 | trimSuffix "-" }}
-+    {{- printf "%s" $name | trunc 63 | trimSuffix "-" }}
-   {{- end }}
+-  {{- end }}
```

Comment:
The word "end" is still here or am I reading this wrong?

Reply:
Its `+` a `-  {{- end }}`, so the patch would remove the end